.github/workflows/ci.yaml

```diff
@@ -80,7 +80,7 @@ jobs:
       - name: Run command
         if: github.event_name == 'pull_request'
         run: |
-          ./scripts/docker_run ./scripts/xtask --commands --scope=commits:1 ${{ matrix.cmd }}
+          ./scripts/docker_run ./scripts/xtask --commands --scope=all ${{ matrix.cmd }}
```

Comment:
I could merge the conditional steps into one, now that they are the same.

Reply:
Yeah it may make sense to do that; if we need to rollback we can revert the entire commit anyways.